notebooks/hypotests/upperlimit_asy_zfit.ipynb

```diff
@@ -344,9 +344,9 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/Users/matthieumarinangeli/anaconda3/envs/tfn2/lib/python3.7/site-packages/hepstats/hypotests/calculators/asymptotic_calculator.py:32: UserWarning: The function <function Space.limit1d at 0x7fd2f30f8a70> may does not return the actual area/limits but rather the rectangular limits. <zfit Space obs=('x',), axes=(0,), limits=(array([[0.1]]), array([[3.]]))> can also have functional limits that are arbitrarily defined and lay inside the rect_limits. To test if a value is inside, use `inside` or `filter`.\n",
+      "/Users/matthieumarinangeli/hepstats/src/hepstats/hypotests/calculators/asymptotic_calculator.py:34: UserWarning: The function <function Space.limit1d at 0x7fc0a390d8c0> may does not return the actual area/limits but rather the rectangular limits. <zfit Space obs=('x',), axes=(0,), limits=(array([[0.1]]), array([[3.]]))> can also have functional limits that are arbitrarily defined and lay inside the rect_limits. To test if a value is inside, use `inside` or `filter`.\n",
       "  bounds = space.limit1d\n",
-      "/Users/matthieumarinangeli/anaconda3/envs/tfn2/lib/python3.7/site-packages/hepstats/hypotests/calculators/asymptotic_calculator.py:37: UserWarning: The function <function Space.area at 0x7fd2f30f8830> may does not return the actual area/limits but rather the rectangular limits. <zfit Space obs=('x',), axes=(0,), limits=(array([[0.1]]), array([[3.]]))> can also have functional limits that are arbitrarily defined and lay inside the rect_limits. To test if a value is inside, use `inside` or `filter`.\n",
+      "/Users/matthieumarinangeli/hepstats/src/hepstats/hypotests/calculators/asymptotic_calculator.py:39: UserWarning: The function <function Space.area at 0x7fc0a390d680> may does not return the actual area/limits but rather the rectangular limits. <zfit Space obs=('x',), axes=(0,), limits=(array([[0.1]]), array([[3.]]))> can also have functional limits that are arbitrarily defined and lay inside the rect_limits. To test if a value is inside, use `inside` or `filter`.\n",
```

Comment:
Can you not run this on Binder to have paths that are not clearly tied to your account? Again, not a big issue.

Reply:
That would require some work to commit notebooks that ran on binder... and I don't think that's a big problem to be honest.

Reply:
Nope, not a problem.